### PR TITLE
fix BENCHMARK_DIR not used

### PR DESCRIPTION
**Pull request by norihiro-w**
fix an issue that benchmarking with the user provided BENCHMARK_DIR is not working

### CODE REVIEW

**Comment by wenqing** [comment]
:+1:

File: scripts/cmake/test/RunBenchmark.cmake [hunk 1]
```diff
@@ -1,4 +1,4 @@
-set(BENCHMARK_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/benchmarks/${benchmarkDir})
+set(BENCHMARK_OUTPUT_DIRECTORY ${BENCHMARK_DIR}/${benchmarkDir})
```
Comment (bilke) [inline scripts/cmake/test/RunBenchmark.cmake]: The intention here is to have benchmark output files just in the build-director to keep the source dir in a clean state. This is especially helpful in Jenkins to have a build and benchmark run in one enclosed directory.

So I suggest to either revert this change or introduce another variable (e.g. `BENCHMARK_OUTPUT_DIR`).

Comment (norihiro-w) [inline scripts/cmake/test/RunBenchmark.cmake]: i see. then i will revert this change. is it still ok if I keep the change at line 50?

Comment (bilke) [inline scripts/cmake/test/RunBenchmark.cmake]: Sure, go ahead!

**Comment by norihiro-w** [comment]
reverted the change and keep the behavior that benchmark outputs are generated under build directories.
